content/hardware/04.pro/carriers/portenta-max-carrier/tutorials/mpcie-4g-modem/content.md

```diff
@@ -62,6 +62,8 @@ The following image provides the position of the Power Jack on the Portenta Max
 
 ![Portenta Max Carrier Power Jack](assets/portentaMAXcarrier_power_distro.png)
 
+***Modems may get stuck on certain occasions, so it is recommended that power be managed through software to allow modem rebooting when necessary.***
```

Comment:
Modems are complex devices, so it is recommended that the user knows each software component involved in order to troubleshoot potential issues.